releases/cloud/point-2023-3-release-note.md

```diff
@@ -0,0 +1,63 @@
+---
+description: >-
+  This article lists new features, improvements, and bug fixes in Syskit Point Cloud
+  version 2023.3.
+---
+
+# Syskit Point 2023.3
+
+[Start a free trial](https://www.syskit.com/products/point/free-trial/) and [tell us what you think.](https://www.syskit.com/company/contact-us/)
+
+**Product version:** 2023.3.0
+
+**Build number:** XXX
+
+**Release date:** July 12, 2023
+
+## About Syskit Point Cloud
+
+* **New to Syskit Point Cloud?** Explore Syskit Point Cloud with a 21-day [free trial](https://www.syskit.com/products/point/free-trial/) for an easy and effective way to manage and secure your environment.
+* **Already using Syskit Point Cloud?** Syskit Point Cloud is automatically upgraded to the latest version when available. The automatic update occurs outside of working hours to ensure minimal interference with your day-to-day business.
+
+## Features
+
+* **Introducing the Policy Automation feature!** Syskit Point Admins can now define rules, which Syskit Point will follow, and, automatically apply defined governance policies for workspaces that meet the conditions configured in the said rules.
+You can fine-tune rules to apply governance policies to workspaces based on:
+  * **Workspace name**
+  * **Privacy**
+  * **Sharing Settings**
+  * **URL**
+  * **Workspace Type**
+  * **Sensitivity Label**
+
+[To learn all about policy automation and how to set it up, follow this link](TODO add link).   
+
+* **Notifications for provisioning errors are now available**. Syskit Point Admins will get an email in case of issues when the Syskit Point service creates a new workspace. Admins can also define custom recipients responsible for addressing such issues when they arise. 
+As an admin, you can expect such emails when: 
+  * **Workspace provisioning failed**
+  * **Workspace provisioning succeeded, but additional workspace configuration resulted in an error**
+  * **Sensitivity label couldn't be applied to a provisioned workspace**
+
+[Learn more about available settings here](TODO add link).
+
+## Improvements & Bug Fixes
```

Comment:
maybe mention the metadata sync issue where it was not always syncing custom metadata? This ways we wont get any emails from UHG :)

Reply:
Please, note that this is a SaaS-only release note, so it doesn't apply to our DC customers. Yet.

Reply:
Will still add it though.

Reply:
custom metadata is synced regardless of cloud or not, and all customers were affected

Reply:
I know, but you're mentioning a DC customer and this is a SaaS release.